.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Awesome, we got another JetBrains user!

Reply:
I'm going to have to update the .gitignore in the future for these though so these files aren't committed.